### ISSUE TITLE
A more recent version for RB960PGS

### ISSUE THREAD

**Issue by leoniDEV** [adron-s/openwrt-rb2011#1]
Can we have a more recent version of OpenWRT for the RB960PGS?
I'm trying to install Jool, but it has compatibility problems with the kernel of 22.03.3

**Comment by n3ph** [comment]
I would like to see ATH79 support for all RB2011 versions to go upstream. 

@adron-s Any plans here?

**Comment by g6094199** [comment]
no. this is a dead end. its better to get rid of our device. :-(

**Comment by n3ph** [comment]
~~I haven't had a chance yet, but [these](https://github.com/adron-s/openwrt-rb2011/commit/aa426e215973136128f3378559b21516b912de38) [commits](https://github.com/adron-s/openwrt-rb2011/commit/90990f1bb9bd8b2c80e96c0ca7afaf227f66cb3a) seem pretty good.~~
~~Still need to build an image for the RouterBOARD 2011UiAS-RM, but this couldn't be that hard..~~ 

**Edit:** I have missed [this](https://forum.openwrt.org/t/wip-porting-mikrotik-rb2011-to-ath79-testers-needed/83309/79) part.

**Comment by mathiasrabe** [comment]
It seems that there are some preparations to make the movement to DSA a little bit more easy: https://github.com/openwrt/openwrt/pull/13589

Maybe there will be some support in the future but we need to be patient.

**Comment by n3ph** [comment]
@leoniDEV Check out `v22.03.5` and cherry-pick all [relevant commits](https://github.com/adron-s/openwrt-rb2011/commits/mikrotik-rb2011)

---
@mathiasrabe [That's](https://github.com/openwrt/openwrt/compare/main...mathiasrabe:openwrt-rb960:rb960) interesting too.. 😉

**Comment by mathiasrabe** [comment]
@n3ph I'm just playing around to get OpenWRT running on my RB960PGS. It compiles successfully but I could not test it yet.

**Comment by leoniDEV** [comment]
@n3ph Sorry, but I'm unable to build the image myself 😳
